src/node_zlib.cc

```diff
@@ -304,10 +314,10 @@ class ZCtx : public AsyncWrap {
 
     HandleScope handle_scope(env->isolate());
     Context::Scope context_scope(env->context());
-
     if (!CheckError(ctx))
       return;
 
+
```

Comment:
Of course, sorry about that. I have a habit of deleting or adding empty lines while reading code.